teradatalabs/cdh5-hive-kerberized/Dockerfile

```diff
@@ -74,11 +74,14 @@ ADD files/conf/hiveserver2-site.xml /etc/hive/conf/hiveserver2-site.xml
 
 # CREATE PRESTO PRINCIPAL AND KEYTAB
 RUN /usr/sbin/kadmin.local -q "addprinc -randkey presto-server/presto-master.docker.cluster@LABS.TERADATA.COM"
+RUN /usr/sbin/kadmin.local -q "addprinc -randkey presto-server/presto-worker.docker.cluster@LABS.TERADATA.COM"
```

Comment:
Maybe the sequence should be `presto-worker-1.docker.cluster`, `presto-worker-2.docker.cluster` and `presto-worker-3.docker.cluster`. It's a little weird to start from `presto-worker.docker.cluster`